draft-ietf-quic-http.md

```diff
@@ -953,9 +953,9 @@ might have acted on.
 
 A client that is unable to retry requests loses all requests that are in flight
 when the server closes the connection.  An endpoint MAY send multiple GOAWAY
-frames indicating different identifiers, but MUST NOT increase the identifier
-value they send, since clients might already have retried unprocessed requests
-on another connection.
+frames indicating different identifiers, but the identifier in each frame
+MUST NOT be greater than the identifier in any previous frame, since clients
+might already have retried unprocessed requests on another connection.
```

Comment:
I like the improvement but will highlight that this is an alternative way to present the described behaviour later on 

```
Like the server, the client MAY send subsequent GOAWAY
frames so long as the specified Push ID is strictly smaller than all previously
sent values.
```